packages/core/src/tracing/instrumentation/loggers/log4js.js

```diff
@@ -34,7 +34,8 @@ function instrumentLog4jsLogger(loggerModule) {
 
 function shimLog(originalLog) {
   return function (level) {
-    if (cls.skipExitTracing({ isActive, skipAllowRootExitSpanPresence: true })) {
+    // The __instana attribute identifies the Instana logger, so prevent these logs from being traced.
+    if (this.__instana || cls.skipExitTracing({ isActive, skipAllowRootExitSpanPresence: true })) {
```

Comment:
Suggestion: Readability

// CASE ....
if (this.__instana) {
}

// CASE: ...
if (cls.skipExitTracing) {}